spec/models/embeddable/labbook_answer_spec.rb

```diff
@@ -42,27 +42,21 @@
     end
   end
 
-  describe '#show_in_runtime? and #show_in_report' do
-    let(:stubs) { { show_in_runtime?: true, show_in_report?: true } }
+  describe '#show_in_runtime?' do
+    let(:stubs) { { show_in_runtime?: true } }
     let(:labbook) { stub_model(Embeddable::Labbook, stubs) }
 
     describe 'with an enabled labbook' do
       it "should return true for show_in_runtime?" do
         expect(labbook_answer.show_in_runtime?).to eql(true)
       end
-      it "should return true for show_in_report?" do
-        expect(labbook_answer.show_in_report?).to eql(true)
-      end
     end
 
     describe 'with a disabled labbook' do
       let(:stubs) { { show_in_runtime?: false, show_in_report?: false } }
       it "should return false for #show_in_runtime?" do
         expect(labbook_answer.show_in_runtime?).to eql(false)
       end
-      it "should return false for #show_in_report?" do
-        expect(labbook_answer.show_in_report?).to eql(false)
-      end
     end
 
```

Comment:
We probably want to test when a lab book is hidden it doesn't have any effect on the students progress.